jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/sam/SamExecutable.kt

```diff
@@ -18,7 +18,7 @@ class SamExecutable : ExecutableType<SemVer>, AutoResolvable, Validatable {
     override val id: String = "samCli"
 
     // inclusive
-    val samMinVersion = SemVer("0.38.0", 0, 38, 0)
+    val samMinVersion = SemVer("0.47.0", 0, 47, 0)
```

Comment:
min version is 47? wasn't it 46?

Reply:
They listed it as 0.47
https://github.com/awslabs/aws-sam-cli/releases

Reply:
I typoed the changelog, fixed it